### PR TITLE
[plugin.audio.radiofrance] 1.0.0

### PR DESCRIPTION
**Pull request by bateast**
Access Radiofrance (French public radio stations) lives steams and podcasts.

see https://www.radiofrance.fr/ and https://www.radiofrance.fr/podcasts.

This is done by parsing the Sieve data (__data.json) of each page and re-creating the page structure.

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project.
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by basrieter** [comment]
Please make the add-on checker happy.

File: plugin.audio.radiofrance/addon.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <import addon="xbmc.python" version="3.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">https://www.radiofrance.fr</summary>
+        <license>GNU General Public License, v3</license>
+        <description lang="en">Get media, livestations and podcasts from RadioFrance (https://www.radiofrance.fr/podcasts)</description>
+        <website>https://github.com/bateast/plugin.audio.podcastrf</website>
```
Comment (basrieter) [inline plugin.audio.radiofrance/addon.xml]: This URL redirects: https://github.com/bateast/plugin.audio.podcastrf redirects to https://github.com/bateast/plugin.audio.radiofrance. Please update it with the correct one.

File: plugin.audio.radiofrance/resources/fanart.svg
Comment (basrieter) [inline plugin.audio.radiofrance/resources/fanart.svg:1]: Please exclude SVG files.

File: plugin.audio.radiofrance/resources/icon.svg
Comment (basrieter) [inline plugin.audio.radiofrance/resources/icon.svg:1]: Please exclude SVG files.

File: plugin.audio.radiofrance/addon.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <requires>
+        <import addon="script.module.requests" version="2.25.1" />
+        <import addon="xbmc.python" version="3.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">https://www.radiofrance.fr</summary>
+        <license>GNU General Public License, v3</license>
```
Comment (basrieter) [inline plugin.audio.radiofrance/addon.xml]: Please use the SPDX identifier: https://spdx.org/licenses/

**Comment by basrieter** [comment]
There is still an issue with the JPG, it now has the executable mark set?

File: plugin.audio.radiofrance/addon.xml (hunk 1 above)
Comment (bateast) [reply plugin.audio.radiofrance/addon.xml]: ok

File: plugin.audio.radiofrance/resources/fanart.svg
Comment (bateast) [reply plugin.audio.radiofrance/resources/fanart.svg:1]: ok

File: plugin.audio.radiofrance/resources/icon.svg
Comment (bateast) [reply plugin.audio.radiofrance/resources/icon.svg:1]: ok

File: plugin.audio.radiofrance/addon.xml (hunk 2 above)
Comment (bateast) [reply plugin.audio.radiofrance/addon.xml]: ok

**Review by basrieter — APPROVED** [review]

**Pull request merged**